scripts/plugins/check_auto_backup_results.py

```diff
@@ -108,12 +109,21 @@ def main():
     # * the download_path musth be valid
     try:
         backup_results = json.loads(raw_backup_results)
+        # "ERROR" will contain a traceback if something crashed during the backups.
+        # See scripts/templates/auto_backup.py::AutoJujuBackupAll.run()
         if "ERROR" in backup_results:
             msg = "Detected error when performing backup: '{}'".format(
                 backup_results["ERROR"]
             )
             nagios_exit(NAGIOS_STATUS_CRITICAL, msg)
 
+        # This entry is populated by the jujubackupall backup process,
+        # and indicates which backups failed. Some backups may have succeeded.
+        if "errors" in backup_results:
+            errors: list[dict[str, Any]] = backup_results["errors"]
+            msg = "Detected error when performing backup: '{}'".format(errors)
```

Comment:
Should we pretty print the error list? Can we output more than one error to nagios, or structured output?